### PR TITLE
BETA: Register soulfulkrishna.is-a.dev

### PR DESCRIPTION
**Pull request by soulfulkrishna**
Added `soulfulkrishna.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/soulfulkrishna.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ecstaticsoul",
+        "email": "soulfulkrishna@gmail.com"
+    },
+    "record": {
+        "URL": "www.soulfulkrishna.is-a.dev"
```
Comment (21Z) [inline domains/soulfulkrishna.json:7]: - `URL` must contain protocol
- You can't set the same domain as `URL`

**Review by 21Z — CHANGES_REQUESTED** [review]

File: domains/kaybee.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ecstaticsoul",
+        "email": "soulfulkrishna@gmail.com"
+    },
+    "record": {
+        "URL": "https://kaybee.is-a.dev"
```
Comment (21Z) [inline domains/kaybee.json:7]: You can't put the same domain for `URL`, put the link of the website you want the to redirect to.

File: domains/krish.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ecstaticsoul",
+        "email": "soulfulkrishna@gmail.com"
+    },
+    "record": {
+        "URL": "https://krish.is-a.dev"
```
Comment (21Z) [inline domains/krish.json:7]: You can't put the same domain for `URL`

File: domains/soulfulkrishna.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ecstaticsoul",
+        "email": "soulfulkrishna@gmail.com"
+    },
+    "record": {
+        "URL": "https://www.soulfulkrishna.is-a.dev"
```
Comment (21Z) [inline domains/soulfulkrishna.json:7]: You can't put the same domain for `URL`

**Review by 21Z — CHANGES_REQUESTED** [review]

**Review by soulfulkrishna — COMMENTED** [review]
Updated two pages. URL redirected to a GitHub page

**Pull request closed without merging**